### PR TITLE
Match regex including group support

### PR DESCRIPTION
**Pull request by slavikm**
Simple utility to match regex in text (or object) including extraction of groups

### CODE REVIEW

File: Scripts/script-MatchRegex.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+script: |-
+  var data = (typeof args.data) === 'string' ? args.data : JSON.stringify(args.data);
+  var flags = args.flags ? args.flags : 'im';
+  var r = new RegExp(args.regex, flags);
+  var m = r.exec(data);
+  if (m && m.length > 0) {
+      var val = m[0];
+      var group = parseInt(args.group);
+      if (args.group && m.length > group) {
+          val = m[group];
```
Comment (meirwah) [inline Scripts/script-MatchRegex.yml]: maybe check here ```m.length > group```

File: Scripts/script-MatchRegex.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      }
+      var ec = {};
+      if (args.out) {
```
Comment (meirwah) [inline Scripts/script-MatchRegex.yml]: will change ```out``` to ```contextKey```, will already use the convention in some places...

File: Scripts/script-MatchRegex.yml (hunk 1 above)
Comment (slavikm) [inline Scripts/script-MatchRegex.yml]: I'm checking in the if, no?

File: Scripts/script-MatchRegex.yml (hunk 2 above)
Comment (slavikm) [inline Scripts/script-MatchRegex.yml]: ok, contextKey it is

**Pull request merged**